platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformBuildOperationIntegrationTest.groovy

```diff
@@ -1153,6 +1153,9 @@ class ArtifactTransformBuildOperationIntegrationTest extends AbstractIntegration
                 @PathSensitive(PathSensitivity.RELATIVE)
                 abstract Provider<FileSystemLocation> getInputArtifact()
 
+                @Inject
+                abstract InputChanges getInputChanges()
```

Comment:
❌ We'd be losing some test coverage here by not checking what happens for non-incremental project transforms.

Reply:
This is the same comment essentially as the one a few lines higher, correct?

Reply:
Yes.